lib/actions.ex

```diff
@@ -226,22 +226,22 @@ defmodule ExTus.Actions do
 
   def get_upload_location(conn, upload_type, identifier) do
     base_url =
-    case Application.get_env(:extus, :environment) do
-      :prod ->
-        scheme = :https
-        ("#{scheme}://#{conn.host }")
+    case conn.host do
+      "api.oktalk.com" ->
```

Comment:
I recently added a config in ok_api after with `Application.get_env(:extus, :environment)` _should_ work, can you please check if it does so that we can remove the hard-coded strings.

https://github.com/bombinatetech/ok_api/blob/master/config/config.exs#L40